assignment1/cube_sum.rb

```diff
@@ -0,0 +1,9 @@
+def sum_of_cubes num_range, power
+    sum = 0
+    num_range.each { |arr|
+        sum += arr ** power
+    }
+    puts sum
+end
+
+sum_of_cubes (1..4), 2
```

Comment:
It is good you have written a dynamic function that will accept any power but please adjust your code for giving problem conditions only also please return the value from the method instead of using "puts".

Reply:
Also please try to write the problem statement in the first line as a comment.